lib/resources/Balance.js

```diff
@@ -9,16 +9,4 @@ module.exports = StripeResource.extend({
   retrieve: stripeMethod({
```

Comment:
The `balance` object is a "singleton", it doesn't have an ID. You just send a GET request to `/v1/balance` and get the singleton object back.

Reply:
yes, it's a singleton retrieve, not a standard retrieve.

Reply:
(that is, it takes no `id`)

Reply:
Any reason this is isn’t `includeBasic: ['retrieve']`?